pom.xml

```diff
@@ -98,6 +98,42 @@
     <checkstyle.configLocation>hawkular-checkstyle/checkstyle.xml</checkstyle.configLocation>
     <checkstyle.consoleOutput>true</checkstyle.consoleOutput>
     <checkstyle.failOnError>true</checkstyle.failOnError>
+    <hawkular.checkstyle.includes>
```

Comment:
Prefixed with "hawkular" because `checkstyle.excludes` is already in use by the plugin itself. It seems that overriding it causes other problems...